Content/Items/Manabonds/ShockingManabond.cs

```diff
@@ -67,6 +70,12 @@ public override void SetStaticDefaults()
 			DisplayName.SetDefault("Shock Bolt");
 		}
 
+		public override void OnSpawn(IEntitySource source)
+		{
+			parent = parentToAssign;
```

Comment:
assuming these are for multiplayer syncing. make sure you add them to sendExtraAI and receiveExtraAI so the data is sent and received